docs/UsingCppInterOp.rst

```diff
@@ -65,35 +65,255 @@ robust (simple function calls, no inheritance, etc.). The goal is to make it as
 close to the compiler API as possible, and each routine should do just one thing.
 that it was designed for.
 
-Further Enhancing the Dynamic/Automatic bindings in CPPYY
-=========================================================
-The main use case for CppInterOp is the CPPYY service. CPPYY is an
-automatic run-time bindings generator for Python and C++, and supports a wide
-range of C++ features (e.g., template instantiation). It operates on demand and
-generates only what is necessary. It requires a compiler (Cling or Clang-REPL).
-that can be available during programme runtime.
+How cppyy leverages CppInterOp
+===============================
+
+CppInterOp supports tools that utilize interactive C++ by using the compiler as
+a service, such as cppyy.
+
+1. **CppInterOp enables interoperability with C++ code**: CppInterOp provides a
+   minimalist and robust interface for language interoperability on the fly,
+   which helps CPPYY generate dynamic Python-C++ bindings by using the a C++
+   interpreter (e.g., Clang-REPL/ Cling) and LLVM.
+
+2. **Reducing dependencies**: Reducing domain-specific dependencies of cppyy to
```

Comment:
go ahead and be specific here - eg "Cling" and "ROOT"

Reply:
updated text:

Reducing domain-specific dependencies of cppyy
   (e.g., on the Cling interpreter and the ROOT framework) to enable more
   generalized usage.